Budget/Add Expense/AddExpenseViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  AddExpenseViewController.swift
+//  Budget
+//
+//  Created by Marijn Schilling on 23/08/2018.
+//  Copyright © 2018 Bakken & Bæck. All rights reserved.
+//
+
+
+import UIKit
+
+protocol AddExpenseViewControllerDelegate: class {
+    func addExpenseViewControllerDidCancel(_ viewController: AddExpenseViewController)
+    func addExpenseViewController(_ viewController: AddExpenseViewController, didAddExpense expense: Int)
+}
+
+class AddExpenseViewController: UIViewController {
+    public weak var delegate: AddExpenseViewControllerDelegate?
+    var amount = 0
+    
+    public var addExpenseView: AddExpenseView! {
+        guard isViewLoaded else { return nil }
+        return view as? AddExpenseView
+    }
+
+    override func viewDidLoad() {
+        addExpenseView.amountTextField.becomeFirstResponder()
+    }
+    
+    @IBAction func didTapDismiss(_ sender: UITapGestureRecognizer) {
+        delegate?.addExpenseViewControllerDidCancel(self)
+        addExpenseView.amountTextField.resignFirstResponder()
+        dismiss(animated: true)
+    }
+
+    @IBAction func didTapAddButton() {
```

Comment:
![img_e799efef1615-1](https://user-images.githubusercontent.com/474246/44656723-4b987180-a9fa-11e8-9fba-69d8ba0520a6.jpeg)

Reply:
Aaah yes, will need to figure out how to do constraint the right way innnn IB

Reply:
Oke that is wayyy to much work. Finding out why not to use IB again 🙈 

It doesnt matter for now since the design is just a prototype. When i implement a real design i will not use IB and make it right for all devices.